packages/contents/src/drivefs.ts

```diff
@@ -24,7 +24,7 @@ import {
 } from './emscripten';
 
 export const DRIVE_SEPARATOR = ':';
-export const DRIVE_API_PATH = '/api/drive.v1';
+export const SERVICE_WORKER_API_PATH = '/sw-api.v1';
```

Comment:
Actually, do we need to keep this in `drivefs.ts`? It looks like it's only exported from this module, and used as the name of the `BroadcastChannel`?